packages/analytics/dashboard-renderer/src/types/dashboard-renderer-types.ts

```diff
@@ -61,42 +81,116 @@ export const gaugeChartSchema = {
 
 export type GaugeChartOptions = FromSchema<typeof gaugeChartSchema>
 
-export const tileSchema = {
+export const tileDefinitionSchema = {
   type: 'object',
   properties: {
     query: {
       // TODO: JSON Schema for Explore v4.
       type: 'object',
     },
     chart: {
-      oneOf: [barChartSchema, gaugeChartSchema],
+      oneOf: [barChartSchema, gaugeChartSchema, timeseriesChartSchema],
     },
-    title: {
+  },
+  required: ['query', 'chart'],
+  additionalProperties: false,
+} as const satisfies JSONSchema
+
+export type TileDefinition = FromSchema<typeof tileDefinitionSchema>
+
+export const tileLayoutSchema = {
+  type: 'object',
+  properties: {
+    position: {
+      type: 'object',
+      properties: {
+        col: {
+          type: 'number',
+        },
+        row: {
+          type: 'number',
+        },
+      },
+      description: 'Position of the tile in the grid.',
+      required: ['col', 'row'],
+      additionalProperties: false,
+    },
+    size: {
+      type: 'object',
+      properties: {
+        cols: {
+          type: 'number',
+        },
+        rows: {
+          type: 'number',
+        },
+      },
+      description: 'Number of columns and rows the tile occupies.',
+      required: ['cols', 'rows'],
+      additionalProperties: false,
+    },
+  },
+  required: ['position', 'size'],
+  additionalProperties: false,
+} as const satisfies JSONSchema
+
+export type TileLayout = FromSchema<typeof tileLayoutSchema>
+
+export const tileConfigSchema = {
+  type: 'object',
+  properties: {
+    id: {
```

Comment:
We had talked about how we should autogenerate the ID rather than requiring it, but I forgot to post my review comment about that earlier -- sorry about that.  Since autogenerating the ID is probably a more substantial change, I'm fine doing it in a followup -- would you mind creating a ticket?